package.json

```diff
@@ -348,6 +353,12 @@
         "mac": "cmd+enter",
         "when": "editorTextFocus && editorLangId == 'rmd'"
       },
+      {
+        "command": "r.runSelectionRetainCursor",
+        "key": "alt+enter",
```

Comment:
I set up the key binding here but I'm not sure why it does not come to be my default key bindings when I run the extension.

Reply:
For me alt+enter worked without needing to change anything. Maybe you have something else bound to alt+enter that has higher priority?